lib/core/proxy.c

```diff
@@ -398,6 +398,52 @@ static void copy_stats(struct rp_generator_t *self)
     self->src_req->proxy_stats.bytes_read.body = self->client->bytes_read.body;
 }
 
+static void copy_ssl_info(h2o_req_t *req, struct st_h2o_socket_ssl_t *ssl)
+{
+    h2o_socket_t s;
+    s.ssl = ssl;
+    req->proxy_stats.ssl.protocol_version = h2o_socket_get_ssl_protocol_version(&s);
+    req->proxy_stats.ssl.session_reused = h2o_socket_get_ssl_session_reused(&s);
+    req->proxy_stats.ssl.cipher = h2o_socket_get_ssl_cipher(&s);;
+    req->proxy_stats.ssl.cipher_bits = h2o_socket_get_ssl_cipher_bits(&s);
+    /* server name and session id are omitted since they are not static data */    
```

Comment:
I no not think we need to repeat the comment here (omitting these values is fine).